### PR TITLE
changing my caption

### PR DESCRIPTION
**Pull request by mike142784**
Changing my caption to hiya.

### CODE REVIEW

**Comment by davidleatham** [comment]
┗|｀O′|┛

**Comment by mike142784** [comment]
(╯°□°）╯︵ ┻━┻

**Comment by NarkTherm** [comment]
No

**Comment by melinate** [comment]
such wow

**Review by NarkTherm — APPROVED** [review]
So approved much good

File: _slides/15-mike142784.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+**Hiya**
```
Comment (melinate) [inline _slides/15-mike142784.md:6]: Nice use of bold

**Review by melinate — APPROVED** [review]

**Pull request merged**